modules/apps/jaggery/emm/src/emm/modules/sso.js

```diff
@@ -119,7 +119,7 @@ var log = new Log();
                 if (sso.responseSign) {
                     if (ssoMod.client.validateSignature(samlRespObj, sso.keyStoreProps)) {
                         var sessionObj = ssoMod.client.decodeSAMLLoginResponse(samlRespObj, samlResponse,
-                            sso.sessionId);
+                                                                               sso.sessionId);
```

Comment:
better breaking the code on line 121 from .decodeSAMLLoginResponse(...) for improved readability